packages/RichText-Tests.package/RTEAddComponentTest.class/instance/testDeleteButtonOfCoreObject.st

```diff
@@ -3,6 +3,4 @@ testDeleteButtonOfCoreObject
 
 	component componentChosen: RTEImageComponent.
 	self click: (self menuItemOf: component coreObject Named: 'deleteComponent').
-	self assert: component submorphs size equals: 1.
-	self assert: component submorphs first equals: component menu.
-	self assert: component coreObject equals: nil.
\ No newline at end of file
+	self assert: ((component submorphs size = 1) and: (component submorphs first = component menu) and: (component coreObject = nil))
```

Comment:
We should discuss with tom if it is better to habe multiple asserts or and them. I like the previous version more but we should vote I think.